lib/client/src/utils/ObjectUtils.js

```diff
@@ -8,39 +8,38 @@
 
 (function(wilson, _) {
 
-
+  
   /**
    * Deletes all object contents
    *
    * @param object
    */
-  var clearObject = function(object) {
+  wilson.utils.clearObject = function clearObject(object) {
     for (var member in object) {
       delete object[member];
     }
   };
 
+  
   /**
    * Replace all @object contents with @newObject properties
    *
    * @param object
    * @param newObject
    */
-  var replaceObject = function(object, newObject) {
-    clearObject(object);
+  wilson.utils.replaceObject = function replaceObject(object, newObject) {
+    wilson.utils.clearObject(object);
```

Comment:
We should probably use an internal function here too for clearObject() to prevent issues with overrides.